### PR TITLE
Upload supporting evidence

### PR DESCRIPTION
**Pull request by gidsg**
This adds a basic pure server-side upload capability.
To test this, go to:
`/project/title`
this will create the initial project and redirect you to the supporting evidence page.

### CODE REVIEW

File: app/controllers/project/project_title_controller.rb [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
-  def project_title
+
+  before_action { @project = Project.new(user: current_user) }
+
+  def show
+  end
+
+  def put
+    @project.update!(project_params)
+    redirect_to "/project/#{@project.id}/support-evidence"
```
Comment (gidsg) [inline app/controllers/project/project_title_controller.rb]: Is there a rails-way to resolve this route @stuartmccoll? I couldn't figure it out so built it manually for now.

File: app/views/project/project_support_evidence/project_support_evidence.html.erb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            <h2 class="govuk-heading-l nlhf-add-item__header__title">Add evidence</h2>
+          </header>
+          <div class="nlhf-add-item__body">
+            <div class="nlhf-add-item__row">
+              <div class="govuk-character-count" data-module="govuk-character-count" data-maxwords="50">
+                <div class="govuk-form-group">
+                  <label class="govuk-label" for="evidence-description">
+                    Describe the evidence you are providing
+                  </label>
+                  <%= f.text_area :evidence_description, value: "" %>
```
Comment (stuartmccoll) [inline app/views/project/project_support_evidence/project_support_evidence.html.erb]: `class: "govuk-textarea"` should be enough to give you the default GOV.UK styling.

File: app/views/project/project_support_evidence/project_support_evidence.html.erb [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          </header>
+          <div class="nlhf-add-item__body">
+            <div class="nlhf-add-item__row">
+              <div class="govuk-character-count" data-module="govuk-character-count" data-maxwords="50">
+                <div class="govuk-form-group">
+                  <label class="govuk-label" for="evidence-description">
+                    Describe the evidence you are providing
+                  </label>
+                  <%= f.text_area :evidence_description, value: "", class: 'govuk-textarea' %>
+                  <span id="evidence-description-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
```
Comment (gidsg) [inline app/views/project/project_support_evidence/project_support_evidence.html.erb]: I couldn't get this to link to the word count, I guess it is related by ID. Maybe one to look at with @PushBackNeil

File: config/routes.rb [hunk 4]
```diff
@@ -24,7 +24,7 @@
     get ':organisation_id/organisation_summary', to: 'organisation_summary#show', as: :organisation_summary_get
   end
 
-  scope "/3-10k", as: :'/3-10k' do
+  scope "/3-10k", as: :three_to_ten_k do
```
Comment (gidsg) [inline config/routes.rb]: this was previously breaking routing paths as it was not symbol safe.

**Pull request merged**